tensorflow/core/common_runtime/next_pluggable_device/next_pluggable_device.cc

```diff
@@ -62,13 +65,16 @@ NextPluggableDevice::NextPluggableDevice(const SessionOptions& session_options,
   }
 
   if (!options.shape_determination_fns.empty()) {
-    device_context_ = core::RefCountPtr<DeviceContext>(
-        new PjRtDeviceContext(options.shape_determination_fns[0]));
+    device_context_ = core::RefCountPtr<DeviceContext>(new PjRtDeviceContext(
+        options.shape_determination_fns[0],
+        /*use_pjrt_tensor_buffer=*/absl::GetFlag(
+            FLAGS_next_pluggable_device_use_pjrt_tensor_buffer)));
```

Comment:
Nit: store the value of `absl::GetFlag(FLAGS_next_pluggable_device_use_pjrt_tensor_buffer)` in a variable to avoid a duplicate line below?

Reply:
hi @swachhandl , I saw FLAGS_next_pluggable_device_use_pjrt_allocator is also used as global variable and in multiple place, it seems it is more clear compared with local variable.

Reply:
Ahh, I meant doing:

```
bool use_pjrt_tensor_buffer = absl::GetFlag(FLAGS_next_pluggable_device_use_pjrt_tensor_buffer);
```

and using `use_pjrt_tensor_buffer` here and below. Not a strong preference, though.